Source/drlg_l1.cpp

```diff
@@ -830,7 +772,7 @@ void PlaceMiniSetRandom(const Miniset &miniset, int rndper)
 		for (int sx = 0; sx < DMAXX - sw; sx++) {
 			if (!miniset.matches({ sx, sy }, false))
 				continue;
-			if (!CanReplaceTile(miniset.replace[0][0], { sx, sy }))
+			if (!CanReplaceTile(miniset.replace[0][0], { { sx, sy }, miniset.size }))

```

Comment:
Feeding this function a rectangle doesn't seem appropriate, plus the PR had a nice self-contained purpose without that commit :P

Reply:
Reverted.